costmap:
  global_frame: map
  robot_base_frame: base_footprint

  update_frequency: 1.0
  publish_frequency: 1.0
  transform_tolerance: 2.0

  static_map: true
  footprint: [[-0.105, -0.105], [-0.105, 0.105], [0.105, 0.105], [0.105, -0.105]]
  static_layer: {enabled: true, lethal_cost_threshold: 60, map_topic: map}
  plugins:
    - {name: static_layer,           type: "costmap_2d::StaticLayer"}
    - {name: inflation_layer,        type: "costmap_2d::InflationLayer"}